starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,10 @@
 // Find the maximum
+var nb1 = 2;
+var nb2 = 3;
+
+function maxOfTwoNumbers(nb1, nb2){
+  return Math.max(nb1, nb2);
+}
 
```

Comment:
Here you don't need to create variables with real numbers at the beginning. You just need to write the function you did after which is perfectly working.